src/roi/__tests__/waterShed.test.ts

```diff
@@ -70,8 +69,8 @@ describe('Test WaterShed Roi generation', () => {
       blackRois: [],
     });
     const rois = roiMapManager.getRois({ kind: 'bw' });
-    expect(rois[0].origin).toEqual({ column: 5, row: 5 });
-    expect(rois[1].origin).toEqual({ column: 2, row: 1 });
```

Comment:
There are too many things that are not tested here. The number of rois and the dimensions of the roi for example.

Reply:
Actually, this should not be tested. Some reason as below, we already checked the roiMapManager is correct.